### ISSUE TITLE
Schnorr and ECDSA extended to non-string arguments

### ISSUE THREAD

**Issue by codygunton** [AztecProtocol/barretenberg#659]
By using `auto`  I extended the native ECDSA and Schorr signatures code to take in not just strings, but byte arrays. Probably using `auto` here without any constraints is unwise. Someone should review (some of this code has been audited externally).

**Comment by federicobarbacovi** [comment]
@ledwards2225 The code referenced by this issue is not part of `bb`. More precisely, the code referenced by this issue is [this](https://github.com/AztecProtocol/aztec-packages/blob/17090383e4d06ce635c4f1b0e6670cc079354e89/circuits/cpp/barretenberg/cpp/src/barretenberg/crypto/ecdsa/ecdsa.hpp#L36) but in `bb` we have [this](https://github.com/AztecProtocol/aztec-packages/blob/be1d142785e80677f73f7655b2ca0bce3129dbab/barretenberg/cpp/src/barretenberg/crypto/ecdsa/ecdsa.hpp#L41).

In short: in `bb` we don't have `auto message`, we have `std::string message`.

Do you have any more info/do you think this issue should be closed?

**Comment by ledwards2225** [comment]
> [@ledwards2225](https://github.com/ledwards2225) The code referenced by this issue is not part of `bb`. More precisely, the code referenced by this issue is [this](https://github.com/AztecProtocol/aztec-packages/blob/17090383e4d06ce635c4f1b0e6670cc079354e89/circuits/cpp/barretenberg/cpp/src/barretenberg/crypto/ecdsa/ecdsa.hpp#L36) but in `bb` we have [this](https://github.com/AztecProtocol/aztec-packages/blob/be1d142785e80677f73f7655b2ca0bce3129dbab/barretenberg/cpp/src/barretenberg/crypto/ecdsa/ecdsa.hpp#L41).
> 
> In short: in `bb` we don't have `auto message`, we have `std::string message`.
> 
> Do you have any more info/do you think this issue should be closed?

Weird, I don't know what that is, some kind of unused fork. Looks like we can close
